### DISCUSSION TITLE
When will be ready 2.0.0 version of the module Microsoft.Graph.DeviceManagement.Enrolment?

### DISCUSSION THREAD

**Discussion by ztrhgf** [microsoftgraph/msgraph-sdk-powershell#2152] [General]
We want to migrate all graph modules to version 2.0.0, but Microsoft.Graph.DeviceManagement.Enrolment is still just 1.28.0. When the version 2.0.0 will be released?

**Comment by mark3grahams** [comment]
I think it is moving to Microsoft.Graph.DeviceManagement.Enrollment fixing the spelling but many CMDlets are missing.

v1.28
(Get-Command -Module Microsoft.Graph.DeviceManagement.Enrolment).count       
390

v2.0
(Get-Command -Module Microsoft.Graph.DeviceManagement.Enrollment).count
24

v2.0 Beta
(Get-Command -Module Microsoft.Graph.Beta.DeviceManagement.Enrollment).count
190
